hist/hist/inc/TGraph.h

```diff
@@ -67,6 +67,7 @@ class TGraph : public TNamed, public TAttLine, public TAttFill, public TAttMarke
 public:
    // TGraph status bits
    enum EStatusBits {
+      kNoStats       = BIT(9),   ///< Don't draw stats box
```

Comment:
Did you check that the `BIT(9)` doesn't conflict in the parent class(es)? I.e. `BIT(9)` could already be used by any of the `TNamed`, `TAttLine`, `TAttFill`, `TAttMarker` classes

Reply:
Yes, I checked it is use only in TH1 for exactly the same thing

Reply:
I looked in  TGraph.cxx the other bits are not initialised in the constructors. There is only two call to ResetBit and that not in the constructors.:
```
hist/src/TGraph.cxx:         const_cast <TGraph*>(this)->ResetBit(kResetHisto);
hist/src/TGraph.cxx:   if (editable) ResetBit(kNotEditable);
```
I guess the BIT are initialised elsewhere. The behaviour I get is correct.i It is backward compatible.